imap_processing/tests/codice/test_codice_l1a.py

```diff
@@ -128,6 +112,11 @@ def test_l1a_data(request) -> xr.Dataset:
     """
 
     dataset = process_codice_l1a(file_path=request.param, data_version="001")
+
+    # Write the dataset to a CDF so it can be manually inspected as well
+    file_path = write_cdf(dataset)
+    print(f"CDF file written to {file_path}")
```

Comment:
```suggestion
```

Reply:
I suppose I should change this to a `logging` statement. I like to have this path printed so that I can copy over the resulting CDF in case I want to manually inspect it.